sdk/src/test/scala/io/horizen/history/validation/ConsensusValidatorOmmersTest.scala

```diff
@@ -40,7 +40,7 @@ class ConsensusValidatorOmmersTest
 
   @Before
   def init(): Unit = {
-    ForkManagerUtil.initializeForkManager(new SimpleForkConfigurator(), "regtest")
+    ForkManagerUtil.initializeForkManager(new Sc2ScOptionalForkConfigurator(), "regtest")
```

Comment:
Why did we change it?

Reply:
My bad, I left it because one of the test was failing, but reverted since the code changed and now it's useless